### ISSUE TITLE
I ran setup and it still says "ModuleNotFoundError: No module named 'requests'

### ISSUE THREAD

**Issue by jellyfamhamz** [zoony1337/TikTok-Shares-Botter#19]

**Comment by zoony1337** [comment]
"— make sure PIP is added to PATH."

**Comment by murdaboyj** [comment]
how can i check if it is. i think that may be the issue because when i open setup it tells me to update it

**Comment by bjames22** [comment]
Yeah I added PIP to PATH and it worked finally. https://appuals.com/fix-pip-is-not-recognized-as-an-internal-or-external-command/ this link heklps do that if u dont know.
